### PR TITLE
Added backend, database to Docker Compose, now we have 2 separate Compose files for staging & production

### PR DESCRIPTION
**Pull request by Saboteur777**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+version: '3.6'
+
+services:
+  frontend:
+    container_name: joladnijo_frontend_${ENVIRONMENT}
+    restart: always
+    expose:
+      - 80
```
Comment (molnarbence) [inline docker-compose.yml:8]: Ha jol sejtem, akkor meg lesz egy nginx service is a teljes stackben, es az fogja proxyzni a frontend service fele a forgalmat. Viszont az expose-nak van meg ertelme? Ugyanazon a networkon ugy tudom, hogy expose nelkul is elerhetoek lesznek a containerek.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    logging:
+      options:
+        max-size: '1m'
+        max-file: '3'
+
+  backend:
+    container_name: joladnijo_backend_${ENVIRONMENT}
+    restart: always
+    expose:
+      - 8000
```
Comment (molnarbence) [inline docker-compose.yml:24]: Itt is ugyanez mint az elozo kommentben, szukseg van meg az expose-ra?

**Comment by Saboteur777** [comment]
A proxy már fent van, működik, de a projekttől teljesen független: https://gitlab.com/webmenedzser/hosting-with-docker

A dokumentációja szerint kell expose-olni: https://github.com/nginx-proxy/nginx-proxy

"The containers being proxied must [expose](https://docs.docker.com/engine/reference/run/#expose-incoming-ports) the port to be proxied, either by using the EXPOSE directive in their Dockerfile or by using the --expose flag to docker run or docker create and be in the same network. By default, if you don't pass the --net flag when your nginx-proxy container is created, it will only be attached to the default bridge network. This means that it will not be able to connect to containers on networks other than bridge."

**Review by molnarbence — APPROVED** [review]
LGTM. Mehet

**Pull request merged**